db/model/reviews.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const faker = require('faker');
+
+mongoose.connect('mongodb://localhost/test');// { useMongoClient: true }
+
+mongoose.Promise = global.Promise;
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('connected to db');
+});
+
+var reviewSchema = new mongoose.Schema({
+  propertyName: String,
+  username: String,
+  avatar: String,
+  review: String,
+  dayPosted: Date,
+  rating: {
+    cleanliness: Number,
+    communication: Number,
+    accuracy: Number,
+    checkIn: Number,
+    location: Number,
+    value: Number
+  },
+  response: {
+    hostname: String,
+    avatar: String,
+    response: String,
+    dayCommented: Date
+  }
+});
+
+var Review = mongoose.model('Review', reviewSchema);
+// var review = mongoose.model( name: review, reviewSchema, collection 'Review');
+
+// module.exports.Review = Review;
+
+var saveData = (data) => {
```

Comment:
I know you mentioned style guide will be implemented later, but just as reminder since it wasn't obvious to me, we have to use let or const, and avoid var.